bin/elastipubsub/main.c

```diff
@@ -158,24 +159,26 @@ static void s_parse_options(int argc, char **argv, struct app_ctx *ctx) {
             case 'h':
                 s_usage(0);
                 break;
+            case 0x02: {
```

Comment:
Super minor question: Should we add a comment saying something like "start of text" or "stx" here? I found it a little confusing initially since the other switch statements are comparing against characters, but then once I searched up what `0x02` is mapped to it made sense, but I'm wondering if perhaps it would be best to add a quick comment just so at a glance it is more obvious what it is looking for.

Reply:
Explicitly added the endpoint argument and instructions